serverless/serverless-kafka.adoc

```diff
@@ -0,0 +1,19 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-kafka"]
+= Using Apache Kafka with {ServerlessProductName}
+include::modules/common-attributes.adoc[]
+:context: serverless-kafka
+
+toc::[]
+
+:FeatureName: Apache Kafka on {ServerlessProductName}
+include::modules/technology-preview.adoc[leveloffset=+2]
+
+You can enable supported Apache Kafka features for use with Knative Eventing, by installing the required Knative Kafka components on your cluster.
+You must also configure the integration between {ServerlessProductName} and a supported link:https://access.redhat.com/documentation/en-us/red_hat_amq/7.6/html/amq_streams_on_openshift_overview/index[Red Hat AMQ Streams] cluster.
+
+Installing these components enables you to use the `KafkaChannel` channel type and `KafkaSource` event source type in your cluster.
```

Comment:
Missed this the first go-around. I think we want to avoid using "enables" in this situation. There are some good use case examples if you look up "allow" and "enable" in the ISG **Appendix C. Word usage**.

s/Installing these components enables you to use the `KafkaChannel` channel type and `KafkaSource` event source type in your cluster./By installing these components, you can use the `KafkaChannel` channel type and `KafkaSource` event source type in your cluster./

Reply:
Did a bit of rewriting here